### DISCUSSION TITLE
Noob needs Help with Github Pages

### DISCUSSION THREAD

**Discussion by driftingS13** [community/community#51624] [New to GitHub] [answered]
Hello, I have been trying to get my project on github pages, i was initially able to get it up but it wasnt scaling like on my vscode live server, so i tried making some changes to my css by adding @media querys for different screen sizes, then when i tried to upload i was having jekyll conversion failure logged in the actions, i did a bunch of changes going through the entire internet trying to figure out what was wrong, including chatgpt, i spent a day and a half trying to get that going with the same failure every time, so i deleted that repo and started another one, now i can get it uploading but my css isnt applying and my pics arnt displaying, now i have spent another day trying to get this going, tried everything from a bunch of different ways to point to my css file, adding and removing assets folder, all sorts of stuff, im just flat out at a loss. i just started learning how to code like a week ago so im really not sure what im doing wrong, or what im reading when trying to troubleshoot it. any help would be appreciated thanks. my repo is here: https://github.com/driftingS13/christhorney.github.io

**Comment by radar-blip** [comment, marked as answer]
In your index.html, I don't think you need the forward slash in front of the folder. So..
Try changing this:
`<link rel="stylesheet" href="/css/style.css" type="text/css">`
to this:
`<link rel="stylesheet" href="css/style.css" type="text/css">`

If this works, do the same for any links containing images. (remove `/` in front of images).

**Reply by driftingS13** [reply]
i will give that a try, i think i added them tho when i was having issues, will report back once the fix goes up and i see what happens

**Reply by driftingS13** [reply]
So it seemes i needed to remove the / from the html and add a ../ to my css, im so confused tho because i had it like this and it didnt work, having said that i have made other changes since then adding and removing a lot of things its hard to keep track. it was probably a combo of things and some of them were probably fixed with chatgpt's help but also chatgpt sent me down rabbit holes like asset folders and adding the / you got me to remove, right now its working thats all that matters thanks so much for your help! literally 3 days of struggling with this instead of learning lol it would have been another 3 days without your help i appreciate it a lot!

**Reply by radar-blip** [reply]
You're welcome. Glad I could help!
